src/Servers/Kestrel/test/FunctionalTests/MaxRequestBufferSizeTests.cs

```diff
@@ -115,7 +115,7 @@ public static IEnumerable<object[]> LargeUploadData
     }
     [Theory]
     [MemberData(nameof(LargeUploadData))]
-    [QuarantinedTest("This is inherently flaky and should never be unquarantined.")]
+    // This is inherently flaky and is relying on helix retry to pass consistently
```

Comment:
I hope this works!

Reply:
So far so good, haven't had any of the retry tests get quarantined yet (knock on wood)